src/store/filestore.rs

```diff
@@ -0,0 +1,209 @@
+use crate::{
+    error::{FileExists, FileHasNoParent, NoAnnotationFound, NoRegexMatch},
+    model::{from_yaml, to_yaml, Pod},
+    store::Store,
+    util::get_struct_name,
+};
+use colored::Colorize;
+use glob::{GlobError, Paths};
+use regex::Regex;
+use std::{collections::BTreeMap, error::Error, fs, iter::Map, path::PathBuf};
+
+#[derive(Debug)]
+pub struct LocalFileStore {
+    pub directory: PathBuf,
+}
+
+impl Store for LocalFileStore {
+    fn save_pod(&self, pod: &Pod) -> Result<(), Box<dyn Error>> {
+        let class = get_struct_name::<Pod>()?;
```

Comment:
What's supposed to be returned here? Wouldn't it always be "Pod"? If so, I find the whole struct name look up an unnecessary step.

Reply:
Save pod is only really meant to return Ok() empty if it is successful. The user should already have a handle of the pod when passing in.

As for get struct_name, we didn't want to hardcode it, hence the get_struct_name function to standardize it.